src/slc/mailrouter/storage.py

```diff
@@ -44,7 +44,7 @@ def lookup(self, uid, _marker=None):
         return self._reverse.get(uid, _marker)
 
     def __getitem__(self, key):
-        return self._forward.items()[key]
+        return list(self._forward.items())[key]
```

Comment:
This looks wrong

Reply:
I suspect that it's not that simple, because if it were then it could already have been done this way in python 2. But I'll look into it.

Reply:
On closer inspection it looks like this method was already broken and is never used.

Reply:
Ah, I found the spot where it is used after all. Will provide an actual fix.

Reply:
Hm. There are two spots in the code that treat the storage as a sequence of tuples. So, it does work like that, and there's not really a straightforward way to make it saner. Maybe I'll come up with something, but I won't give it highest priority.

Reply:
Oh I thought this one was changed. It seems to me we either remove it or cover with a test because I cannot see how this could work.